### PR TITLE
Deploy configuration files to EC2 instance

### PR DESCRIPTION
**Pull request by alyoshenka**

### CODE REVIEW

File: .github/workflows/deployment.yml [hunk 1]
```diff
@@ -17,18 +17,19 @@ jobs:
     runs-on: ubuntu-latest
     if: ${{ github.event.workflow_run.conclusion == 'success' }}
     steps:
+      - uses: actions/checkout@v3
```
Comment (alyoshenka) [inline .github/workflows/deployment.yml:20]: Fun fact! This is why it wasn't working...I didn't actually check out the repo in the workflow.

**Comment by alyoshenka** [comment]
Fixes #111

**Review by LucasKnezevich — APPROVED** [review]
👍

**Review by VictorHugoNava — APPROVED** [review]
LGTM

**Pull request merged**